src/main/protobuf/extvm/msg.proto

```diff
@@ -0,0 +1,142 @@
+syntax = "proto3";
+package io.iohk.ethereum.extvm;
+
+message CallContext {
+  bytes ownerAddr = 1;
+  bytes callerAddr = 2;
+  bytes originAddr = 3;
+  bytes contractCode = 4;
+  bytes inputData = 5;
+  bytes callValue = 6;
+  bytes gasPrice = 7;
+  bytes gasProvided = 8;
+  int32 callDepth = 9;
+  bytes receivingAddr = 10;
```

Comment:
one more field will have to be added here, a string, containing the name of the function that is the entry point of the transaction on the contract that is being called. That field should be empty on transactions which create a new contract.

Reply:
That's doable, the EVM can simply ignore that.. but how would Mantis ever provide anything relevant in that field? I guess we would have to modify the transaction object, but that may complicate things if we want to maintain support for Ethereum in the same codebase.